### PR TITLE
SUMO-212665: Fix terraform resource refresh interval

### PR DESCRIPTION
**Pull request by charlessumo**

### CODE REVIEW

File: sumologic/resource_sumologic_dashboard.go [hunk 1]
```diff
@@ -79,7 +79,7 @@ func resourceSumologicDashboard() *schema.Resource {
 			"refresh_interval": {
 				Type:         schema.TypeInt,
 				Optional:     true,
-				ValidateFunc: validation.IntInSlice([]int{120, 300, 900, 1800, 3600, 7200, 86400}),
+				ValidateFunc: validation.IntInSlice([]int{0, 120, 300, 900, 1800, 3600, 7200, 86400}),
```
Comment (sumovishal) [inline sumologic/resource_sumologic_dashboard.go:82]: The API has a different set of values https://api.sumologic.com/docs/#operation/createDashboard. Which ones are correct?

Comment (charlessumo) [reply sumologic/resource_sumologic_dashboard.go:82]: We enforce these on the backend, should use this set
"2m", "5m", "15m", "30m", "1h", "2h", "1d"

Comment (charlessumo) [reply sumologic/resource_sumologic_dashboard.go:82]: Can you approve this and I'll make changes to the dashboard api docs?

**Review by sumovishal — APPROVED** [review]
lgtm, please fix the API docs too.

**Comment by sumovishal** [comment]
merging the PR. The failing tests are unrelated.

**Pull request merged**